src/Provision.js

```diff
@@ -608,6 +606,17 @@ const CreateFabricObject = async ({client, libraryId, typeId, publicMetadata, t}
     metadata: publicMetadata
   });
 
+  if (privateMetadata){
```

Comment:
`if (foo) {` not `if (foo){`

seems lots of folks' IDEs aren't helping with this; wayne was able to set up a linter for himself as he kept having the same problem, maybe we need a .eslintrc.json